src/components/Dashboard/Analytics/AnalyticsUtils.js

```diff
@@ -0,0 +1,121 @@
+// Barcharts
+
+export const barChartOptions = {
+    plugins: {
+        legend: {
+            position: "top",
+            display: false,
+        },
+        title: {
+            display: false,
+            text: "Activity",
+        },
+    },
+    maintainAspectRatio: false,
+    responsive: true,
+};
+
+export const mainBarChartDatasets = [
+    {
+        label: "Contributions days",
+        data: [345, 567, 765, 453, 676, 876, 564, 234, 440, 280, 440, 280],
+
+        borderColor: "rgb(128, 128, 128)",
+        backgroundColor: [
```

Comment:
you could write here function for random color. you need something like this because you hard coded too much colors without meaning